.github/workflows/release.yml

```diff
@@ -0,0 +1,54 @@
+name: Release
+run-name: Release v${{ inputs.version }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        description: |
+          Specify the version to release.
+          The version should be in `X.X.X`-format, following semver.
+        required: true
+
+jobs:
+  CI:
+    uses: ./.github/workflows/gh-workflow.yml
+    with:
+      version: ${{ inputs.version }}
```

Comment:
Perhaps verify the version is of a valid format.

Reply:
Good suggestion, will look into it